addons/binding/org.openhab.binding.harmonyhub/README.md

```diff
@@ -70,7 +70,15 @@ items:
 String HarmonyGreatRoomActivity              "Current Activity [%s]"  (gMain) { channel="harmonyhub:hub:GreatRoom:currentActivity" }
 ```
 
-Devices can send button presses
+Hubs can also send a button press to a device associated with the current activity. A String item can be used to send any button name/label or a Player item can be used to send Play/Pause/FastForward/Rewind/SkipForward/SkipBackward. This mimics the physical remote where buttons are mapped differently depending on which activity is running.  For example the play button may be sent to a DVD player when running a DVD activity, or it may be sent to a AppleTV when running that activity. 
```

Comment:
It's easier to add comments in the README's when each sentence starts on a new line.

Reply:
good to know.